src/EditorFeatures/CSharpTest/ImplementAbstractClass/ImplementAbstractClassTests.cs

```diff
@@ -1911,5 +1911,59 @@ public override void AbstractMethod()
     }
 }", parseOptions: TestOptions.RegularPreview);
         }
+
+        [WorkItem(48742, "https://github.com/dotnet/roslyn/issues/48742")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsImplementAbstractClass)]
+        public async Task TestUnconstrainedGenericNullable()
+        {
+            await TestAllOptionsOffAsync(
+@"abstract class B<T>
+{
+    public abstract T? M();
+}
+
+class [|D|] : B<int>
+{
+}",
+@"abstract class B<T>
+{
+    public abstract T? M();
+}
+
+class D : B<int>
+{
+    public override int M()
+    {
+        throw new System.NotImplementedException();
+    }
+}");
+        }
+
+        [WorkItem(48742, "https://github.com/dotnet/roslyn/issues/48742")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsImplementAbstractClass)]
+        public async Task TestUnconstrainedGenericNullable2()
+        {
+            await TestAllOptionsOffAsync(
+@"abstract class B<T>
+{
+    public abstract T? M();
+}
+
+class [|D<T>|] : B<T> where T : struct
+{
+}",
+@"abstract class B<T>
+{
+    public abstract T? M();
+}
+
+class D<T> : B<T> where T : struct
+{
+    public override T M()
+    {
+        throw new System.NotImplementedException();
+    }
+}");
+        }
```

Comment:
can we a few more tests.  

1. where T is unconstrained
2. where T is constrained to `class`.  
3. where T is unconstrained, and passed as T? to the base type
3. where T is constrained to `class`, and passed as T? to the base type
3. where T is constrained to `struct`, and passed as T? to the base type

Reply:
Sure, make sure that other cases doesn't regress.